aziotctl/src/config/mp.rs

```diff
@@ -0,0 +1,100 @@
+// Copyright (c) Microsoft. All rights reserved.
+
+//! This subcommand takes a connection string and writes out the super-config file
+//! configured to use manual provisioning with that connection string.
+//! All other settings are left as their defaults.
+
+use anyhow::{anyhow, Context};
+
+use aziotctl_common::config as common_config;
+
+#[derive(structopt::StructOpt)]
+pub(crate) struct Options {
+    /// The connection string
+    #[structopt(short = "c", long, value_name = "CONNECTION_STRING")]
+    connection_string: String,
+
+    /// The path of the system configuration file to write to
+    #[structopt(
+        short = "o",
+        long,
+        value_name = "FILE",
+        default_value = "/etc/aziot/config.toml"
+    )]
+    out_config_file: std::path::PathBuf,
+
+    /// Overwrite the new configuration file if it already exists
+    #[structopt(short = "f", long, value_name = "FILE")]
```

Comment:
Should the `value_name` be something else? Like `FORCE`?

Reply:
Oops. Fixed. (It's not supposed to have a `value_name` since it's a flag.)